src/AWS.Deploy.CLI/ServerMode/Controllers/DeploymentController.cs

```diff
@@ -116,7 +130,7 @@ public async Task<IActionResult> GetRecommendations(string sessionId)
 
             var output = new GetRecommendationsOutput();
 
-            state.NewRecommendations = await orchestrator.GenerateDeploymentRecommendations();
+            state.NewRecommendations ??= await orchestrator.GenerateDeploymentRecommendations();
```

Comment:
above `NewRecommendations` is replaced with newly available recommendations but here only if it is null. A comment/description would help why it is done like this.